server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -551,7 +552,7 @@ export class ScribeLambda implements IPartitionLambda {
 		this.pendingP.then(
 			() => {
 				this.pendingP = undefined;
-				this.context.checkpoint(queuedMessage);
+				this.context.checkpoint(queuedMessage, this.restartOnCheckpointFailure);
```

Comment:
I wonder if we should do this for deli and scriptorium too. I think I saw a few similar failures there as well

Reply:
Updated